src/components/hero.js

```diff
@@ -0,0 +1,187 @@
+import { useStaticQuery } from "gatsby"
```

Comment:
Wasnt sure where to put this but we probably want the bg to be page specific. So we would pass the value into this component as a prop.

Reply:
Passed this in as a child, as you said on stream from what I gathered.